oci_python_image/py_image_layer.bzl

```diff
@@ -43,6 +43,14 @@ def py_image_layer(name, binary, root = None, **kwargs):
 
     runfiles(
         name = "%s/app/runfiles" % name,
+        include = "**/*",
+        exclude = "**/*python*-linux*/**",
+        **runfiles_kwargs
+    )
+
+    runfiles(
+        name = "%s/python/runfiles" % name,
```

Comment:
python/runfiles makes me think this layer stores the interpreter, but this is the third-party packages in here right?

Reply:
no, it's the interpreter. we would have a third layer for 3p packages. should I add that as well?

Reply:
Added you on a Bazel Slack thread about this

Reply:
@thesayyn Can you please add the third layer for third-party PyPI packages? I guess it will be pretty simple, I just struggle with correct regex pattern.